### PR TITLE
Adding play icon on mobile

### PR DESCRIPTION
**Pull request by noppanit**
#43

### CODE REVIEW

File: inc/class-gif2html5.php [hunk 1]
```diff
@@ -434,7 +435,7 @@ public function unset_conversion_response_pending( $attachment_id ) {
 	public function filter_the_content_img_to_video( $html ) {
 		wp_enqueue_script( 'gif2html5-video-handler' );
 		wp_enqueue_script( 'gif2html5' );
-
+		wp_enqueue_style( 'gif2html5', plugins_url( 'css/gif2html5.css', dirname( __FILE__ ) ), array(), '0.1-alpha' );
```
Comment (noppanit) [inline inc/class-gif2html5.php]: Will fix the version once this is merged 
https://github.com/fusioneng/gif2html5-plugin/pull/45/files#diff-cc41e4a4a00a4989dc6cf55ae440bee3R24

File: css/gif2html5.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@media only screen and (max-width: 760px) {
+
+	.gif2html5-video-container:before {
+		content: ' ';
+		background-image: url('http://www.clipartbest.com/cliparts/bTy/EkL/bTyEkLGrc.png');
```
Comment (noppanit) [inline css/gif2html5.css]: Not sure what icon we should use. I'm using this one for testing.

Comment (danielbachhuber) [inline css/gif2html5.css]: @lisaej Preference? Here's the original issue https://github.com/fusioneng/gif2html5-plugin/issues/43

Comment (lisaej) [inline css/gif2html5.css]: I think we can use the play icon we use everywhere. If you use .video-link class, it'll apply it to a :before pseudo-element. You'll have to re-position it for the gif2html video to be centered.

Comment (noppanit) [inline css/gif2html5.css]: can we extract that from `fusion-theme`? It's a plugin so it's supposed to be dependent from the theme.

Comment (lisaej) [inline css/gif2html5.css]: Oh crap, I keep forgetting it that it's a standalone plugin. Forget what I said.

**Comment by noppanit** [comment]
Depends on https://github.com/fusioneng/fusion-theme/pull/2967

File: css/gif2html5.css (hunk 2 above)
Comment (danielbachhuber) [inline css/gif2html5.css]: How about a default for the plugin, then an override for our theme?

Comment (lisaej) [inline css/gif2html5.css]: screen shot 2015-05-28 at 8 31 02 pm

We can add an override from here by adding the class.

Comment (noppanit) [inline css/gif2html5.css]: So what would be the default for the plugin? I'm not sure I can just grab anything from the internet or should I just extract that from the theme.

Comment (danielbachhuber) [inline css/gif2html5.css]: > I'm not sure I can just grab anything from the internet

Fontawesome has some good options http://fortawesome.github.io/Font-Awesome/icons/#video-player

You can also pull from Noun Project, but make sure to respect the license.

**Comment by noppanit** [comment]
Here's the screeshots from browserstack

screenshot 2015-05-29 14 35 43

screenshot 2015-05-29 14 31 02

**Comment by noppanit** [comment]
@danielbachhuber #reviewmerge

**Pull request merged**